updatesnap/SnapModule/snapmodule.py

```diff
@@ -394,6 +401,9 @@ def _is_gitlab(self, repository):
         """ Evaluates the URI of a repository and returns an URI
             object with it, but only if it is a Gitlab URI. """
         uri = self._get_uri(repository, 3)
+        # Check for gitlab instance used by debian
```

Comment:
Good point, but maybe this should be in a separate MR.

Reply:
Ok, I will create a separate PR for this